### ISSUE TITLE
AIO: how can we learn what is new in Angular and docs?

### ISSUE THREAD

**Issue by kukjevov** [angular/angular#17409]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
[x] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
```

**Current behavior**
Currently when you open http://angular.io docs pages, you can see documentation for Angular 4.0, but not for Angular 4.2, or any newer version. Does not matter whether i talking about API docs, or guides or anything else. Actually you can not see, that there are some updates in documentation either. That means if i looked at documetation of Animations lets say 4 months ago, how can i know, that there is something new if i open it today? As far as i see, there are some new features in CHANGELOG for Animations
>animations: introduce a wave of new animation features (16c8167)
>animations: introduce routeable animation support (f1a9e3c)

but going through source code and finding what they are is not best way how to find out what is new. Especially if you dont know original code. 

I love your work, but it looks like that there is no time for documentation. Which is crucial for common users, who dont want to crawl your code. 

**Expected behavior**
I would like to see something like *What is new* similar to typescript, which exactly tells you what is new, not only CHANGELOG, which is good as informative material, but you know nothing from it. I know it gives you more work, but how can developers use new features if they dont know what they are.

Anyway thanks :)

**Comment by wardbell** [comment]
Changed the PR title from "Missing documentation for new features" to make clear that this issue asks for a way to learn what's new and is not specifically concerned about the lack of documentation for the new features themselves.

**Comment by kukjevov** [comment]
Well there should be maybe one more thing at the end or start of every page. And that is last date of update, or version for which was article valid. I think this should go along with information what is new :).

Thank you

**Comment by jenniferfell** [comment]
docsarea=currency
docsarea=updating

Thanks. Now that v6 is released, we're getting around to closing some open loops. I apologize for letting this sit. 

In v6 we introduced https://angular.io/guide/updating. It refers to the blog posts for announcements of new features, and those announcements link to the docs for the new features. We'll try to remember to also mention any major documentation changes that are not tied to code changes. 

Regarding documentation explicitly about what's new and changed in the doc--outside of CHANGELOG.md--it looks like @wardbell (?) started to address this request. (Awesome!) Before I saw this issue and discussion, I opened an issue on the need to finish resolving the info arch and doc process for this content (aka where does it go and how do we keep it current). Related issue https://github.com/angular/angular/issues/23615
